src/vs/workbench/api/node/extHostSCM.ts

```diff
@@ -310,7 +310,7 @@ export class ExtHostSCM {
 		return sourceControl;
 	}
 
-	$provideOriginalResource(sourceControlHandle: number, uri: URI): TPromise<URI> {
```

Comment:
I also wasn't entirely sure of `URI` vs `vscode.Uri`. I think the change itself is correct, but I was not entirely sure if any existing code is relying on the old signature or if this could cause any trouble